src/Prefixer.js

```diff
@@ -34,7 +34,14 @@ export function prefix (value, length) {
 			return WEBKIT + value + replace(value, /(\w+).+(:[^]+)/, WEBKIT + 'box-$1$2' + MS + 'flex-$1$2') + value
 		// align-self
 		case 5443:
-			return WEBKIT + value + MS + 'flex-item-' + replace(value, /flex-|-self/, '') + value
+			return WEBKIT + value + MS + 'flex-item-' + replace(value, /flex-|-self/, '')
+				+ (
+					// center, end, start, stretch
+					match(substr(value, length + 1, sizeof(value) - 1), /^(center|end|start|stretch)$/)
```

Comment:
flexbox supports plain `center|end|start|stretch` wouldn't this now add grid properties to what was intended as flexbox `align-self` given that `center|end|start|stretch` are universal properties for both flexbox and grid?

Reply:
Well, at least it is also how autoprefixer handle `align-self`:

```css
.example {
    align-self: stretch
}
```

will become

```css
/*
* Prefixed by https://autoprefixer.github.io
* PostCSS: v8.3.6,
* Autoprefixer: v10.3.1
* Browsers: since 2010
*/

.example {
    -webkit-align-self: stretch;
        -ms-flex-item-align: stretch;
                -ms-grid-row-align: stretch;
            align-self: stretch
}
```